docs/index.rst

```diff
@@ -617,12 +617,9 @@ A standard ``bool`` field will raise a ``ValidationError`` if the value is not o
   ``'off', 'f', 'false', 'n', 'no', '1', 'on', 't', 'true', 'y', 'yes'``
 * a ``bytes`` which is valid (per the previous rule) when decoded to ``str``
 
-For stricter behavior, ``StrictBool`` can be used to require specifically ``True`` or ``False``;
```

Comment:
Can we keep this line and add a link to the "Strict Types" section.

Sorry to mess you around.